client/src/constants/russian.ts

```diff
@@ -0,0 +1,133 @@
+  task_form_submit: 'Сохранить задачу',
+  task_table_description: 'Описание',
+  task_table_done: 'Сделанный',
+  task_table_actions: 'Действия',
+  task_table_action_done: 'Сделанный',
+  task_table_action_undone: 'Отменено',
+  task_table_action_done_yes: 'Да',
+  task_table_action_done_no: 'Нет',
+  task_table_action_edit: 'Редактировать',
+  task_table_action_delete: 'Удалить',
+
+  note_form_title: 'Добавить примечание',
+  note_form_title_label: 'Заголовок',
+  note_form_title_placeholder: 'Введите название заметки',
+  note_form_content_label: 'Примечание к содержанию',
+  note_form_content_placeholder: 'Введите содержание заметки',
+  note_form_submit: 'Сохранить заметку',
+  note_table_btn_edit: 'Редактировать',
+  note_table_btn_delete: 'Удалить',
+
+  about_app_title: 'О TaskNote',
+  about_app_description: `TaskNote — это ваше приложение для управления
+    задачами и заметками в одном удобном месте. Независимо от того, отслеживаете ли вы
+    ежедневные задачи или организуете заметки с важных встреч, TaskNote помогает
+    вам оставаться организованными и продуктивными.`,
+  about_app_features: 'Функции',
+  about_app_features_one: 'Быстро добавляйте и управляйте задачами и заметками',
+  about_app_features_two: 'Поиск и фильтрация заметок для легкого доступа',
+  about_app_features_three: 'Интуитивно понятный и понятный пользовательский интерфейс',
+  about_app_help_title: 'Помощь и как использовать',
+  about_app_help_description: `Чтобы начать, просто зарегистрируйтесь или войдите в систему, и
+    вы получите доступ к своей персонализированной панели. Оттуда вы можете
+    создавать, редактировать и удалять задачи и заметки, а также организовывать их так, как вам
+    нравится. Нужна помощь? Посетите нашу страницу «Справка» (в будущем) для получения руководств
+    и часто задаваемых вопросов.`,
+
+  about_tech_title: 'Технологии',
+  about_tech_description: `TaskNote был создан с использованием современных веб-технологий, 
+      которые обеспечивают скорость, надежность и безопасность.`,
+  about_tech_list_one: 'React с TypeScript для фронтенда',
+  about_tech_list_two: 'Bootstrap 5 для компонентов и адаптивного дизайна',
+  about_tech_list_three: 'Java и Spring Boot плюс GraalVM для бэкэнда и Cloud Native',
+  about_tech_list_four: 'PostgreSQL для управления базами данных',
+  about_tech_list_five: 'Docker для контейнеризации и развертывания',
+  about_tech_list_six: 'Действия GitHub для CI/CD, тестирования и принудительного линтинга',
+  about_tech_list_seven: 'SonarCloud и GitHub QL для проверок безопасности и улучшений',
+
+  about_dev_title: 'О разработчике',
+  about_dev_description: `Привет! Я Рикардо, разработчик TaskNote. Я увлечен созданием приложений,
+    которые делают жизнь проще и более
+    организованной. Вы можете связаться со мной по адресу `,
+  about_dev_description_two: ' для любых вопросов или отзывов.',
+
+  account_my_account_tittle: 'Мой счет',
```

Comment:
```suggestion
  account_my_account_tittle: 'Мой аккаунт',
```

This is a mistake.
Or "Моя учётная запись" if you like it more. I'd stick with "аккаунт".

Reply:
Done here https://github.com/ricardo-campos-org/react-typescript-todolist/pull/189/commits/12b10e04de236beb52c8d9e180fc721ef5c1ff6e